pkg/utils/leaderelection/leaderelection.go

```diff
@@ -0,0 +1,33 @@
+package leaderelection
+
+import (
+	"context"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/config/clusterstatus"
+	"github.com/openshift/library-go/pkg/config/leaderelection"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+)
+
+func GetLeaderElectionConfig(restcfg *rest.Config, enabled bool) configv1.LeaderElection {
```

Comment:
go linter will cause the build to fail.
please add a comment in the form of:
// GetLeaderElectionConfig ....

Reply:
Oh that's great, thanks! Updated with comments